source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -28,108 +28,94 @@ for a complete list.
 Sample Documents
 ~~~~~~~~~~~~~~~~
 
-The following examples use a database called ``test`` and collection
-called ``pizza`` which contains documents that describe customers and
-their pizza shop orders as array elements in field called ``items``. Use
-the following sample document to follow the example queries:
+The following examples use the ``test.pizza`` collection. The collection
+contains documents that describe the customers and their pizza orders
+as array elements in field called ``items``. The following examples
+update this sample document:
 
 .. code-block:: javascript
 
    [{
       name: "Steve Lobsters",
       address: "731 Yexington Avenue",
       items: [
-        {
-          type: "beverage",
-          name: "Water",
-          size: "17oz"
-        },
-        {
-          type: "pizza",
-          size: "large",
-          toppings: ["pepperoni"],
-        },
-        {
-          type: "pizza",
-          size: "medium",
-          toppings: ["mushrooms", "sausage", "green peppers"],
-          comment: "Extra green peppers please!",
-        },
-        {
-          type: "pizza",
-          size: "large",
-          toppings: ["pineapple, ham"],
-          comment: "red pepper flakes on top",
-        },
-        {
-          type: "calzone",
-          fillings: ["canadian bacon", "sausage", "onion"],
-        },
-        {
-          type: "beverage",
-          name: "Diet Pepsi",
-          size: "16oz",
-        },
+        { type: "beverage", name: "Water", size: "17oz", },
+        { type: "pizza", size: "large", toppings: ["pepperoni"], },
+        { type: "pizza", size: "medium", toppings: ["mushrooms", "sausage", "green peppers"], comment: "Extra green peppers please!", },
+        { type: "pizza", size: "large", toppings: ["pineapple, ham"], comment: "red pepper flakes on top", },
+        { type: "calzone", fillings: ["canadian bacon", "sausage", "onion"], },
+        { type: "beverage", name: "Diet Pepsi", size: "16oz", },
       ],
     },
     {
       name: "Popeye",
       address: "1 Sweethaven",
       items: [
-        {
-          type: "pizza",
-          size: "large",
-          toppings: ["garlic, spinach"],
-        },
-        {
-          type: "calzone",
-          toppings: ["ham"],
-        },
+        { type: "pizza", size: "large", toppings: ["garlic, spinach"], },
+        { type: "calzone", toppings: ["ham"], },
       ],
     }]
 
+Specifying Array Elements
+-------------------------
+
+You can specify which array elements to update using a positional
+operator. Positional operators can specify the first, all, or certain
+array elements to update. 
+
+To specify elements in an array with positional operators, use **dot
+notation**. Dot notation is a property access syntax for navigating BSON
+objects. To learn more, see :manual:`dot notation
```

Comment:
issue: If there is a ref on this page, use it instead. If not, feel free to dismiss this comment.

Reply:
Found their page [here](https://github.com/mongodb/docs/blob/master/source/reference/operator/update-array.txt), there's no ref